src/main/java/org/apache/accumulo/examples/helloworld/InsertWithBatchWriter.java

```diff
@@ -20,46 +20,42 @@
 import org.apache.accumulo.core.client.AccumuloSecurityException;
 import org.apache.accumulo.core.client.BatchWriter;
 import org.apache.accumulo.core.client.Connector;
-import org.apache.accumulo.core.client.MultiTableBatchWriter;
-import org.apache.accumulo.core.client.MutationsRejectedException;
 import org.apache.accumulo.core.client.TableExistsException;
 import org.apache.accumulo.core.client.TableNotFoundException;
 import org.apache.accumulo.core.data.Mutation;
 import org.apache.accumulo.core.data.Value;
-import org.apache.accumulo.examples.cli.BatchWriterOpts;
-import org.apache.accumulo.examples.cli.ClientOnRequiredTable;
 import org.apache.hadoop.io.Text;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 /**
  * Inserts 10K rows (50K entries) into accumulo with each row having 5 entries.
  */
 public class InsertWithBatchWriter {
 
-  public static void main(String[] args) throws AccumuloException, AccumuloSecurityException, MutationsRejectedException, TableExistsException,
+  private static final Logger log = LoggerFactory.getLogger(InsertWithBatchWriter.class);
+
+  public static void main(String[] args) throws AccumuloException, AccumuloSecurityException, TableExistsException,
       TableNotFoundException {
-    ClientOnRequiredTable opts = new ClientOnRequiredTable();
-    BatchWriterOpts bwOpts = new BatchWriterOpts();
-    opts.parseArgs(InsertWithBatchWriter.class.getName(), args, bwOpts);
 
-    Connector connector = opts.getConnector();
-    MultiTableBatchWriter mtbw = connector.createMultiTableBatchWriter(bwOpts.getBatchWriterConfig());
+    Connector connector = Connector.builder().usingProperties("conf/accumulo-client.properties").build();
 
-    if (!connector.tableOperations().exists(opts.getTableName()))
-      connector.tableOperations().create(opts.getTableName());
-    BatchWriter bw = mtbw.getBatchWriter(opts.getTableName());
+    if (!connector.tableOperations().exists("hellotable")) {
+      connector.tableOperations().create("hellotable");
+    }
 
-    Text colf = new Text("colfam");
-    System.out.println("writing ...");
-    for (int i = 0; i < 10000; i++) {
-      Mutation m = new Mutation(new Text(String.format("row_%d", i)));
-      for (int j = 0; j < 5; j++) {
-        m.put(colf, new Text(String.format("colqual_%d", j)), new Value((String.format("value_%d_%d", i, j)).getBytes()));
+    try (BatchWriter bw = connector.createBatchWriter("hellotable")) {
+      Text colf = new Text("colfam");
+      log.trace("writing ...");
+      for (int i = 0; i < 10000; i++) {
+        Mutation m = new Mutation(new Text(String.format("row_%d", i)));
```

Comment:
Can now pass Strings to Mutation, used to could not.  Changing this could be a separate PR.

Reply:
I will update the PR

Reply:
Also created #12

Reply:
Fixed in 2ac71cb726f